internal/workflow/processing.go

```diff
@@ -97,7 +97,7 @@ func (w *ProcessingWorkflow) cleanup(ctx temporalsdk_workflow.Context, state *wo
 			UUID:        state.sip.uuid,
 			Name:        state.sip.name,
 			AIPUUID:     state.aip.id,
-			CompletedAt: state.aip.storedAt,
+			CompletedAt: temporalsdk_workflow.Now(dctx).UTC(),
```

Comment:
It looks like we set "CompletedAt" even on error? 

I guess that "StartedAt" and "CompletedAt" are referring to the ingest workflow start and end times? I don't know what it would mean for a SIP to be "completed" in the context of Enduro.  Maybe we should drop these fields from SIP entity at some point in the future.